src/launch/main.c

```diff
@@ -1532,7 +1540,7 @@ static int run(void) {
                 return MAIN_FAILED;
         }
 
-        r = manager_run(manager, !strcmp(main_arg_scope, "system"));
+        r = manager_run(manager, main_arg_audit);
```

Comment:
No reason to propagate that value, anymore, right? `manager_run_child()` should use it directly, imho. We always make sure our commandline arguments are constant globals, so no harm, I think.